GNUmakefile

```diff
@@ -0,0 +1,35 @@
+TEST?=$$(go list ./... | grep -v 'vendor')
+GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
+
+default: build
+
+build: fmtcheck
+	go install
+
+fmt:
+	@echo "==> Fixing source code with gofmt..."
+	gofmt -s -w $(GOFMT_FILES)
```

Comment:
Can we add make command for tidy as well?

Reply:
That's a good idea. (I honestly thought I had done this, I must have reverted the change or be misremembering.)